packages/radix-ui-themes/src/components/base-dialog.css

```diff
@@ -18,7 +18,12 @@
 }
 
 .rt-BaseDialogScrollPadding {
-  margin: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
Previous styles weren't encouraging the dialog content to expand to its full width

Reply:
Compare the dialogs
https://themes-playground-modulz.vercel.app/test-inset
https://themes-playground-git-vlad-a-bag-of-fixes-modulz.vercel.app/test-inset